controllers/apply/form-router-next/lib/form-model.js

```diff
@@ -250,7 +250,7 @@ class FormModel {
         return this.sections[currentSectionIndex + 1];
     }
 
-    previousPage({ baseUrl, sectionSlug, currentStepIndex = null }) {
+    previousPage({ baseUrl, sectionSlug, currentStepIndex = null, copy = {} }) {
```

Comment:
For the form model I'd be inclined to pass the locale to the form model as a whole, or when constructing the form model rather than as a function argument. But that might be overkill for now so I reckon we go with this and revisit.